scripts/CMSRunAnalysis.py

```diff
@@ -165,7 +173,8 @@ def logCMSSW():
     if not os.path.exists("cmsRun-stdout.log"):
         print("ERROR: Cannot dump CMSSW stdout; perhaps CMSSW never executed (e.g.: scriptExe was set)?")
         logCMSSWSaved = True
-        open('logCMSSWSaved.txt', 'a').close()
+        with open('logCMSSWSaved.txt', 'a', encoding='utf-8') as fh:
+            fh.close()
         os.utime('logCMSSWSaved.txt', None)
```

Comment:
Apparently, here we just need to touch an empty file. I do not think we need to change it, but if we really want to please pylint, then we could use

```python
        with open('logCMSSWSaved.txt', 'a', encoding='utf-8'): pass
        os.utime('logCMSSWSaved.txt')
```

- i do not think that we need to pass "None" to os.utime, https://docs.python.org/3/library/os.html#os.utime . I would avoid amiguous arguments.
- I would avoid to explicitly execute commands that should already been taken care by the context manager, so i would avoid calling `fh.close()`